packages/node-cli/src/commands/project/deploy/DeployAction.js

```diff
@@ -70,17 +75,51 @@ module.exports = class DeployAction extends BaseAction {
 				flags.push(COMMAND.FLAGS.APPLY_CONTENT_PROTECTION);
 			}
 
-			const sdkParams = CommandUtils.extractCommandOptions(params, this._commandMetadata);
+			if (params[COMMAND.FLAGS.PREVIEW]) {
+				try {
+					delete params[COMMAND.FLAGS.PREVIEW];
+					flags = flags.slice((0, 2));
```

Comment:
There's extra parenthesis here.

Reply:
I need this extra parenthesis for slice to work properly.